app/assets/stylesheets/components/_navigation.scss

```diff
@@ -2,7 +2,10 @@
 .primary-navigation {
   display: flex;
   flex-wrap: wrap;
-  background-color: $color-true-black;
```

Comment:
This is a design question for @veganstraightedge 

do we want to add a change like this:
```diff
diff --git a/app/assets/stylesheets/components/_navigation.scss b/app/assets/stylesheets/components/_navigation.scss
index e3a0c2be..689318c3 100644
--- a/app/assets/stylesheets/components/_navigation.scss
+++ b/app/assets/stylesheets/components/_navigation.scss
@@ -2,9 +2,17 @@
 .primary-navigation {
   display: flex;
   flex-wrap: wrap;
-  background-color: $color-true-black;
+  animation: colorfader linear infinite;
+  animation-duration: 270s;
+  animation-iteration-count: infinite;
+
+  @media screen and (max-width: $small-tablet) {
+    background-color: $color-true-black;
+  }
 }
 
+@include color-fader("colorfader", $header-color-list);
+
 .nav-link {
   font-size: 1.7rem;
   letter-spacing: .05em;
```

so that when the screen is a small display and the nav-header is on 2 lines the top and bottom row both cycle?

e.g.

only top row             |  both rows
:-------------------------:|:-------------------------:
<img width="738" alt="screen shot 2018-02-15 at 13 48 44" src="https://user-images.githubusercontent.com/13190980/36274840-05cbc408-1257-11e8-8d9b-fd530b9f791e.png"> | <img width="738" alt="screen shot 2018-02-15 at 13 42 29" src="https://user-images.githubusercontent.com/13190980/36274852-165154c8-1257-11e8-89c2-54789f7f499b.png">

Reply:
Good catch @astronaut-wannabe. :shipit:

Reply:
Nice! I didn't even think about that.

Reply:
Is that tweak something you're adding to the mix, @astronaut-wannabe? Just want to make sure I'm not dropping the ball.

Reply:
@TheConObs sorry, I totally missed your question!

I was suggesting that you add that change to this PR

Reply:
Ah OK. Upon some initial low-level tinkering this morning, it will take some tweaking but I'll have this up this weekend.